schema_utils.py

```diff
@@ -387,3 +388,61 @@ def is_valid_email(obj):
             bool. Whether the given object is a valid email.
         """
         return bool(re.search(r'^[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}$', obj))
+
+    @staticmethod
+    def is_valid_math_expression(obj, algebraic=True):
+        """Checks if the given obj (a string) represents a valid algebraic or
+        numeric expression.
+
+        Args:
+            obj: str. The given expression.
+            algebraic: bool. True if the given expression is algebraic
+                else numeric.
```

Comment:
Can you also add: "Note that purely-numeric expressions are NOT considered valid algebraic expressions."

Reply:
Done.